tools/tox_tools/dose_responses/dose_response.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="dr_curve" name="Dose Response Curve for Toxicological Risk Assessement" version="0.1.0+galaxy0">
+    <description>Toxicity prediction tool -  Dose response Curve</description>
+    <creator>
+        <organization name="Helmholtz Centre for Environmental Research - UFZ, Department of Ecotoxicology"
+                      url ="https://www.ufz.de/index.php?en=34241"/>
+    </creator>
+    <requirements>
+        <requirement type="package" version="3.0_1">r-drc</requirement>
+        <requirement type="package" version="4.3.3">r-base</requirement>
+        <requirement type="package" version="3.5.1">r-ggplot2</requirement>
+    </requirements>
+    <command detect_errors="aggressive">
+        <![CDATA[
+        Rscript $__tool_directory__/dose_response.R
+            '$input_csv'
+            '$concentration_column'
+            '$response_column'
+            '$plot_output'
+            '$ec_output'
+        ]]>
+    </command>
+    <inputs>
+        <param name="input_csv" type="data" format="csv" label="Input CSV file"/>
```

Comment:
In general tab separated tables (`tabular`) are preferred in Galaxy. There are also converters (and I think csv is auto-converted).